Bryant_Trinh/Function Basic 2.py

```diff
@@ -0,0 +1,39 @@
+#1
+def function(cd):
+	arr=[]
+	for i in range(cd, 0):
+		arr.pop(i)
+	return arr
+print(arr)
+
+#2
+def function(x,y):
+	print(x)
+	return(y)
+function(x,y)
+
+#3
+sum=0
+def function(arr):
+	sum= arr[0]+len(arr)
+	return sum
+print(sum)
+
+#4
+count=0
+new=[]
+def function(arr):
+	for i in arr(0,len(arr)):
+		if len(arr)<2:
+			return false
+		else if arr[i]>arr[1]:
+			count+=1
+			new.pop(arr[i])
+function(arr)
+
+#5
+def function lengthandvalue(x,y):
+	for i in lengthandvalue(0,x):
```

Comment:
create an empty array before the loop and push values into that array in the for loop